docs/pywbemclicommands.rst

```diff
@@ -1154,15 +1154,66 @@ pywbemcli:
       log: None
 
 
+This feature can be used to modify parameters of a connection beyond the scope of the
+current connection using the general options.  Setting connection parameters using
+general options with the select command causes the changes to become the new current
+connection definition.
+
+For example:
+
+
+.. code-block:: text
+
+    $ pywbemcl --server http://localhost
+    pywbemcli> connection list
+    WBEM server connections: (#: default, *: current)
+    +------------+-------------+-------------+--------+-----------+----------+
+    | name       | server      | namespace   | user   |   timeout | verify   |
+    |------------+-------------+-------------+--------+-----------+----------|
+    | *not-saved | http://blah | root/cimv2  |        |        30 | True     |
+    +------------+-------------+-------------+--------+-----------+----------+
+
+    pywbemcli> --user fred --password 234 connection select
+    pywbemcli> connection list
+    WBEM server connections: (#: default, *: current)
+    +------------+-------------+-------------+--------+-----------+----------+
+    | name       | server      | namespace   | user   |   timeout | verify   |
+    |------------+-------------+-------------+--------+-----------+----------|
+    | *not-saved | http://blah | root/cimv2  | Ron    |        30 | True     |
+    +------------+-------------+-------------+--------+-----------+----------+
+
+    TODO: this feature not done.
+
+
 See :ref:`pywbemcli connection select --help` for details.
 
 
 .. _`Connection show command`:
 
 Connection show command
-^^^^^^^^^^^^^^^^^^^^^^^^^
-The ``connection show`` command shows information in the current connection.  See the the ``select``
-above for an example of this command.
+^^^^^^^^^^^^^^^^^^^^^^^
+The ``connection show`` command shows information about the current connection or the
+connection with name NAME.  The connection to be displayed will be
+the :ref:`current connection` if the NAME argument is omitted, the connection named
+with the NAME argument, or selected from a selection list the NAME argument "?"
```

Comment:
We should not introduce multiple different ways to trigger the prompting. In other commands, the prompting is triggered by the use if the `--interactive` option, and I think that's what we should use also in this case.

Reply:
This clearly show 3 options. Current, defined name or showme.  With the interactive option, you
have an inherent conflict since the name and the -i option can both occur so you have to define what overrides what. 

Actually  I would like to get the interactive option with the instance get/references/associators commands (i.e. try to get with just what is presented (the class) and when that fails do the selection) but you can't because dmtf combined class and instance associators/references that the operation with just the class is valid. 

**DISCUSSION**

Reply:
In the existing cases of --interactive we also specify constraints on the argument: These are 6 cases that are all on instance operations that operate on a single instance, and they all have an INSTANCENAME argument, which is required to be the class name instead of an instance name if the option is used. In the case here, we can require that NAME is omitted if --interactive is used.

Reply:
I opened issue https://github.com/pywbem/pywbemtools/issues/349 for this, so we can move ahead with this PR for now.